src/hexapod_control.cpp

```diff
@@ -117,6 +117,9 @@ int main(int argc, char** argv) {
     hexapod = hebi::Hexapod::create(params, hex_errors);
   }
   std::cout << "Found robot -- starting control program.\n";
+  if (hexapod && !hexapod->setGains() && !hexapod->setGains()) {
```

Comment:
technically, `hexapod` will always be true b/c of the `while` above, but I like the safe programming here.

Reply:
dangers of copy/paste :)